.golangci.yml

```diff
@@ -41,6 +51,16 @@ issues:
     linters:
     - errcheck
     - gosec
+    - forbidigo
+  - path: pkg/cosign/env
+    linters:
+    - forbidigo
+  - path: pkg/providers/
```

Comment:
I actually think most of these are worth documenting. IMO the goal of `cosign env` is to ask "what from my environment is possibly affecting Cosign" or to ask "what knobs do I have to affect Cosign's behavior?"

(Same for `pkg/cosign/git/gitlab`.)

WDYT?

(I think it's reasonable to address that in a later PR, before closing the accompanying issue.)

Reply:
> I actually think most of these are worth documenting. IMO the goal of cosign env is to ask "what from my environment is possibly affecting Cosign" or to ask "what knobs do I have to affect Cosign's behavior?"

@znewman01 Just to make sure I have a clear understanding -- do you mean that I should also add those variables to the slice of known/registered variables (`environmentVariables` in the `env` package)? As a result of that, `cosign env` is also going to print variables like `GITHUB_TOKEN`, `GITLAB_TOKEN`, etc...

In that case, we have to remove checking if the variable starts with `COSIGN_` (https://github.com/sigstore/cosign/pull/2322#discussion_r991726750), but I think it makes sense to do that.

Reply:
Precisely, I think you should add them all to `environmentVariables`.

(And yes, I think the `COSIGN_` prefix check doesn't make sense anymore. Sorry I asked you to add it and now am asking you to take it away!)

Reply:
Fixed.